### PR TITLE
docs: add edit url to merge confidence

### LINKED ISSUE #188: Add custom `edit_url` for the Merge Confidence page
### What would you like to be able to do?

The pencil/edit button on the [Merge Confidence page in our docs](https://docs.renovatebot.com/merge-confidence/) doesn't point to the correct location.

The documentation build pulls in the `README.md` file from [`whitesource/merge-confidence`](https://github.com/whitesource/merge-confidence). This means our default "edit button link" doesn't work for this page. We can fix this by using the new `edit_url` hook.

### Did you already have any implementation ideas?

Put this front matter in the code that we use to load in the `README.md` from the `whitesource/merge-confidence` repository:

```markdown
---
edit_url: https://github.com/whitesource/merge-confidence/edit/main/README.md
---
```

We should probably update these lines of code:

https://github.com/renovatebot/renovatebot.github.io/blob/a6b1afca75d89eeb4846d8b461575685abc31f44/bin/get-docs.sh#L25-L26

### Do you want to write this feature yourself?

No

### PR DESCRIPTION
**Pull request by viceice**
## Changes:
add edit url to merge confidence

## Context:
- closes #188

### CODE REVIEW

File: bin/get-docs.sh [hunk 1]
```diff
@@ -22,5 +22,8 @@ cp -R tmp/docs/* "$docs"
 cd "$pwd"
 cp -R src/* docs/
 
-printf -- '---\ntitle: Merge Confidence\ndescription: Node versions support in Renovate\n---\n\n' > docs/merge-confidence.md
+printf -- '---\n' > docs/merge-confidence.md
+printf -- 'title: Merge Confidence\ndescription: Node versions support in Renovate\n' >> docs/merge-confidence.md
+printf -- 'edit_url: https://github.com/whitesource/merge-confidence/edit/main/README.md\n' >> docs/merge-confidence.md
+printf -- '---\n\n' >> docs/merge-confidence.md
```
Comment (HonkingGoose) [inline bin/get-docs.sh:28]:
```suggestion
printf -- '---\n' > docs/merge-confidence.md
{
  printf -- 'title: Merge Confidence\ndescription: Node versions support in Renovate\n'
  printf -- 'edit_url: https://github.com/whitesource/merge-confidence/edit/main/README.md\n'
  printf -- '---\n\n'
  curl -sSLf https://raw.githubusercontent.com/whitesource/merge-confidence/main/README.md 
} >> docs/merge-confidence.md
```

This should match the shellcheck recommendations. Please check if this works in your preview.

File: bin/get-docs.sh [hunk 2]
```diff
@@ -22,5 +22,11 @@ cp -R tmp/docs/* "$docs"
 cd "$pwd"
 cp -R src/* docs/
 
-printf -- '---\ntitle: Merge Confidence\ndescription: Node versions support in Renovate\n---\n\n' > docs/merge-confidence.md
+{
+  printf -- '---\n'
+  printf -- 'title: Merge Confidence\ndescription: Node versions support in Renovate\n'
```
Comment (HonkingGoose) [inline bin/get-docs.sh:27]: The description doesn't match the page content.

It should probably say something like "Learn about Renovate's Merge Confidence feature" or something similar?

Comment (viceice) [reply bin/get-docs.sh:27]:
```suggestion
  printf -- 'title: Merge Confidence\ndescription: Learn about Renovate's Merge Confidence feature\n'
```

@rarkins WDYT?

File: bin/get-docs.sh [hunk 3]
```diff
@@ -22,5 +22,11 @@ cp -R tmp/docs/* "$docs"
 cd "$pwd"
 cp -R src/* docs/
 
-printf -- '---\ntitle: Merge Confidence\ndescription: Node versions support in Renovate\n---\n\n' > docs/merge-confidence.md
+{
+  printf -- '---\n'
+  printf -- 'title: Merge Confidence\ndescription: Learn about Renovate's Merge Confidence feature\n'
+  printf -- 'edit_url: https://github.com/whitesource/merge-confidence/edit/main/README.md\n'
+  printf -- '---\n\n'
```
[the remark this answers is not in the document]
Comment (HonkingGoose) [reply bin/get-docs.sh:29]: Starting Gitpod on your branch gives this error:

```sh
bin/get-docs.sh: line 29: unexpected EOF while looking for matching `''
```

[the remark this answers is not in the document]
Comment (viceice) [reply bin/get-docs.sh:29]: fixed

[the remark this answers is not in the document]
Comment (viceice) [reply bin/get-docs.sh:29]: your suggwestion wasn't properly escaped 😅

**Pull request merged**